DigitalOcean.API/Clients/IKeysClient.cs

```diff
@@ -20,9 +20,9 @@ public interface IKeysClient {
         Task<Key> Get(object keyIdOrFingerprint);
 
         /// <summary>
-        /// Update an existing key in your account
+        /// To update the name of an SSH key.
         /// </summary>
-        Task<Key> Update(object keyIdOrFingerprint, Models.Requests.Key key);
+        Task<Key> Update(object keyIdOrFingerprint, Models.Requests.UpdateKey updateKey);
```

Comment:
The only thing that can be updated on a Key currently is the name. Explicitly created `UpdateKey` class to separate from class used on Create, `Models.Requests.Key`.